opencv_dnn/cpp/detect-image.cpp

```diff
@@ -32,7 +32,7 @@ int main(int argc, char* argv[]) {
     float nms_thresh = 0.3;
     int keep_top_k = 750;
     // Result
-    bool vis = false;
+    bool vis = true;
```

Comment:
You can make it read from `argv`. Setting this to true by default is not good for compability since we cannot assume users want to visualize and their environment is ready (GTK is required and X11 needed to be enable over SSH).

Reply:
Thank you for your advice! I've added an optional argument so that visualization is manageable.